prov/verbs/src/verbs_cq.c

```diff
@@ -36,6 +36,7 @@
 
 #include "fi_verbs.h"
 
+static uint64_t fi_ibv_comp_flags(struct ibv_wc *wc);
```

Comment:
It should be set to yourself, indicating that you agree to the terms given in the [CONTRIBUTORS file](https://github.com/ofiwg/libfabric/blob/master/CONTRIBUTORS).